index.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable quote-props */
+/* eslint-disable no-use-before-define */
+import fetch from 'node-fetch';
+
+import xlsx from 'xlsx';
+
+import inquirer from 'inquirer';
+
+import dotenv from 'dotenv';
+
+// import SearchItem from './SearchItem';
+
+dotenv.config();
+class SearchItem {
+
+  constructor(title, description, link) {
+    this.title = title;
+    this.description = description;
+    this.link = link;
+  }
+
+}
+
+let searchName = (await inquirer.prompt({
+  name: 'search_name',
+  type: 'input',
+  message: 'What do you want to search in google?', 
+})).search_name;
+
+let searchNumber = (await inquirer.prompt({
+  name: 'search_number',
+  type: 'input',
+  message: 'How many search results do you want? (max 100)', 
+})).search_number;
+
+const search = `q=${searchName}&num=${searchNumber}`;
+// convert to async await
+// fetch(`https://google-search3.p.rapidapi.com/api/v1/search/${search}`, {
+//   method: 'GET',
+//   headers: {
+//     'x-rapidapi-key': process.env.API_KEY,
+//     'x-proxy-location': 'US',
+//   },
+// })
+//   .then((res) => res.json())
+//   .then((res) => mapToSearchItems(res))
+//   .then((searchItems) => toExcelArrayOfArray(searchItems))
+//   .then((arrayOfArray) => createExcel(arrayOfArray))
+//   .catch((error) => console.log('error', error));
+
+async function loadSearchResults() {
+  const response = await fetch(`https://google-search3.p.rapidapi.com/api/v1/search/${search}`, {
+    method: 'GET',
+    headers: {
+      'x-rapidapi-key': process.env.API_KEY,
+      'x-proxy-location': 'US',
+    }, 
+  });
+  const jsonResponse = await response.json();
+  const searchItems = await mapToSearchItems(jsonResponse);
+  const arrayOfArray = await toExcelArrayOfArray(searchItems);
+  await createExcel(arrayOfArray);
+}
+await loadSearchResults();
```

Comment:
dont see why we need to put all this in a method, why not jst leave it bare in script ? just write a comment before the block of code that you are searching the result n etc...